core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -108,8 +109,15 @@ public KafkaStore(SchemaRegistryConfig config,
         JaasUtils.isZkSecurityEnabled());
     this.brokerSeq = zkUtils.getAllBrokersInCluster();
 
-    this.bootstrapBrokers = KafkaStore.getBrokerEndpoints(
-            JavaConversions.seqAsJavaList(this.brokerSeq));
+    List<String> bootstrapServersConfig = config.getList(SchemaRegistryConfig.KAFKASTORE_BOOTSTRAP_SERVERS_CONFIG);
+    List<String> endpoints;
+    if (bootstrapServersConfig.isEmpty()) {
+      endpoints = brokersToEndpoints(JavaConversions.seqAsJavaList(this.brokerSeq));
+    } else {
+      endpoints = config.getList(SchemaRegistryConfig.KAFKASTORE_BOOTSTRAP_SERVERS_CONFIG);
```

Comment:
Probably shouldn't call `getList` again since we already have `bootstrapServersConfig`.